### PR TITLE
Javascript javascript1 week1/gayathri

### PR DESCRIPTION
**Pull request by GayathriVenkatraman**

### CODE REVIEW

File: javascript/javascript1/week1/Ez Namey/startup_name.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  "Link",
+  "Pulse",
+  "Path",
+  "Wave",
+  "Flow",
+  "Bridge",
+  "Tech",
+];
+
+const randomNumber = Math.floor(Math.random() * 10);
```
Comment (LucasIversen) [inline javascript/javascript1/week1/Ez Namey/startup_name.js]: We can use two random numbers to get truly random names :) Right now the names will always come in parallel.

File: javascript/javascript1/week1/Housey pricey/price_estimator.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+const peterGardenSizeInM2 = 100;
+
+const peterHouseActualPrice = 2500000;
+
+const peterHouseExpectedPrice =
+  peterHouseVolumeInMeters * 2.5 * 1000 + peterGardenSizeInM2 * 300;
+
+if (peterHouseActualPrice < peterHouseExpectedPrice) {
+  console.log("Peter house price " + peterHouseExpectedPrice + " is too much");
+} else {
```
Comment (LucasIversen) [inline javascript/javascript1/week1/Housey pricey/price_estimator.js:22]: Hmmm.. What if Peter payed the exactly right price 🤔

File: javascript/javascript1/week1/Housey pricey/price_estimator.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  peterHouseDepthInM2 * peterHouseHeightInM2 * peterHouseWidthInM2;
+const peterGardenSizeInM2 = 100;
+
+const peterHouseActualPrice = 2500000;
+
+const peterHouseExpectedPrice =
+  peterHouseVolumeInMeters * 2.5 * 1000 + peterGardenSizeInM2 * 300;
+
+if (peterHouseActualPrice < peterHouseExpectedPrice) {
+  console.log("Peter house price " + peterHouseExpectedPrice + " is too much");
```
Comment (LucasIversen) [inline javascript/javascript1/week1/Housey pricey/price_estimator.js]: seems like we need a calculation here. Like 

whatPeterPaid - whatThePriceIs

File: javascript/javascript1/week1/Housey pricey/price_estimator.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+const juliaHouseVolumeInMeters =
+  juliaHouseDepthInM2 * juliaHouseHeightInM2 * juliaHouseWidthInM2;
+const juliaGardenSizeInM2 = 70;
+
+const juliaHouseActualPrice = 1000000;
+
+const juliaHouseExpectedPrice =
+  juliaHouseVolumeInMeters * 2.5 * 1000 + juliaGardenSizeInM2 * 300;
+
+if (juliaHouseActualPrice < juliaHouseExpectedPrice) {
```
Comment (LucasIversen) [inline javascript/javascript1/week1/Housey pricey/price_estimator.js:43]: Same thing as above

**Review by LucasIversen — CHANGES_REQUESTED** [review]
Hi Gayathri. Good solution, just needs a few changes :)

File: javascript/javascript1/week1/Ez Namey/startup_name.js (hunk 1 above)
Comment (GayathriVenkatraman) [reply javascript/javascript1/week1/Ez Namey/startup_name.js]: Updated the code with two random numbers

File: javascript/javascript1/week1/Housey pricey/price_estimator.js (hunk 4 above)
Comment (GayathriVenkatraman) [reply javascript/javascript1/week1/Housey pricey/price_estimator.js:43]: Added both actual and estimated price in the code

File: javascript/javascript1/week1/Housey pricey/price_estimator.js (hunk 3 above)
Comment (GayathriVenkatraman) [reply javascript/javascript1/week1/Housey pricey/price_estimator.js]: Added both actual and estimated price in the code

File: javascript/javascript1/week1/Housey pricey/price_estimator.js (hunk 2 above)
Comment (GayathriVenkatraman) [reply javascript/javascript1/week1/Housey pricey/price_estimator.js:22]: Updated the code with the conditon

**Comment by GayathriVenkatraman** [comment]
Hi Lucas,

Thanks your valuable feedback and for your time on reviewing the code. I have updated the code based on your comments. Hence closing the pull request.

Regards,
Gayathri

**Pull request closed without merging**